db/schema.rb

```diff
@@ -70,6 +70,15 @@
   add_index "deploy_groups_stages", ["deploy_group_id"], name: "index_deploy_groups_stages_on_deploy_group_id", using: :btree
   add_index "deploy_groups_stages", ["stage_id"], name: "index_deploy_groups_stages_on_stage_id", using: :btree
 
+  create_table "deploy_response_urls", force: :cascade do |t|
+    t.integer  "deploy_id",    limit: 4,   null: false
+    t.string   "response_url", limit: 255, null: false
+    t.datetime "created_at"
```

Comment:
you can change the migration and schema ... ideally change the migration
and run VERSION=1211221212112 rake db:migrate:down db:migrate:up

On Sun, Jul 17, 2016 at 3:20 PM, Ben Straub <notifications@github.com>
wrote:

> In db/schema.rb
> <https://github.com/zendesk/samson/pull/1130#discussion_r71090876>:
>
> > @@ -70,6 +70,15 @@
> >    add_index "deploy_groups_stages", ["deploy_group_id"], name: "index_deploy_groups_stages_on_deploy_group_id", using: :btree
> >    add_index "deploy_groups_stages", ["stage_id"], name: "index_deploy_groups_stages_on_stage_id", using: :btree
> >
> > +  create_table "deploy_response_urls", force: :cascade do |t|
> > +    t.integer  "deploy_id",    limit: 4,   null: false
> > +    t.string   "response_url", limit: 255, null: false
> > +    t.datetime "created_at"
>
> These were generated by t:timestamps in the migration
> <https://github.com/zendesk/samson/blob/ben/slash-command/plugins/slack_app/db/migrate/20160711222417_create_deploy_response_url_table.rb#L7>.
> Is it standard to override this once it's generated?
>
> —
> You are receiving this because you are on a team that was mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/zendesk/samson/pull/1130/files/d008e855296f051e83d429a560e4e921b94fd917#r71090876>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAAsZ7Asucn3EOyUI22GFvK_D-VEcIuGks5qWqqrgaJpZM4JIRHk>
> .
>